Misc/NEWS.d/next/Library/2024-07-21-10-45-24.gh-issue-122081.dNrYMq.rst

```diff
@@ -0,0 +1 @@
+Fixed crash in optional :func:`!decimal.IEEEContext()` function.
```

Comment:
```suggestion
Fix a crash in the :func:`!decimal.IEEEContext()` optional function
available via the ``EXTRA_FUNCTIONALITY`` configuration flag.
```

We should also document decimal.IEEEContext and `EXTRA_FUNCTIONALITY` as well but in another PR. Perhaps first document them and then move to your PR.